features/support/env.rb

```diff
@@ -11,6 +11,9 @@
 
 require "cucumber/rails"
 
+World DocumentHelper
+World PersonHelper
```

Comment:
Nice one 👍 the commit message here could do with some formatting so that it fits onto one line in the GitHub UI. Also worth explaining why these are being added, to save someone having to navigate to the PR description.

Reply:
No worries, I'll tidy it all up now. I realise now the PR title as well is a bit out of date.

Reply:
Hopefully everything looks a bit neater